AwsCryptographicMaterialProviders/dafny/AwsCryptographyKeyStoreAdmin/Model/KeyStoreAdmin.smithy

```diff
@@ -101,10 +100,34 @@ union KMSIdentifier {
   then those two ARNs may differ in region,
   although they must be otherwise equal.
   If either ARN is not an MRK ARN, then
-  kmsMRKeyArn behaves exactly as kmsKeyArn.")
+  KmsMRKeyArn behaves exactly as kmsKeyArn.")
   KmsMRKeyArn: String,
 }
 
+@documentation(
+"Include all attributes of an item as Encryption Context
+in a KMS Encrypt or Decrypt call,
+effectively signing the attributes.")
+structure KmsAes {
```

Comment:
when i create a KMS key. I create a KMS Symmetric key. Although the names are analogous, I'm worried this _could_ confuse customers and say, "I have a symmetric kms key, do I need a KmsAes key?"

Reply:
I will change this to be `KmsSymmetricEncryptionKey`.

Reply:
Rather, I will change it to `KmsSymmetricEncryption`, 
and the Identifier I will change to `KmsSymmetricEncryptionKey`.